contracts/Tokenomics.sol

```diff
@@ -664,15 +661,6 @@ contract Tokenomics is TokenomicsConstants {
             _codePerDev = codePerDev;
         }
 
-        // Check the epsilonRate value for idf to fit in its size
-        // 2^64 - 1 < 18.5e18, idf is equal at most 1 + epsilonRate < 18e18, which fits in the variable size
-        // epsilonRate is the part of the IDF calculation and thus its change will be accounted for in the next epoch
-        if (_epsilonRate > 0 && _epsilonRate <= 17e18) {
-            epsilonRate = uint64(_epsilonRate);
-        } else {
-            _epsilonRate = epsilonRate;
-        }
```

Comment:
why was this dropped?

Reply:
`epsilonRate` was used to set the default addition to the discount factor set by the DAO. In a new IDF approach, its value varies from 1 to 2 only (whereas in the old approach it could go up to 17), and thus no defaults are needed.